R/translate-sql-odbc.R

```diff
@@ -35,7 +50,79 @@ base_odbc_agg <- sql_translator(
 base_odbc_win <- sql_translator(
   .parent = base_win,
   n             = function() sql("COUNT(*)"),
-  count         = function() sql("COUNT(*)")
+  count         = function() sql("COUNT(*)"),
+
+  # Window functions in this variant use field that its being
+  # 'windowed' as the secondary ORDER BY instead of the field
+  # from win_current_order().  In the tests, having an ORDER BY
```

Comment:
I had to solve this problem in implyr, and I did it by implementing a recursive function that's called in `sql_build()`. See https://github.com/ianmcook/implyr/commit/08a704a130104582c4b531817fd4016980f65114.